addons/binding/org.openhab.binding.pioneeravr/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,123 @@
+	<thing-type id="ipAvr">
+		<label>Pioneer AVR over IP</label>
+		<description>Control a Pioneer AVR over through a Serial port </description>
+
+		<channels>
+			<channel id="power" typeId="powerChannel" />
+			<channel id="volumeDimmer" typeId="volumeChannelDimmer" />
+			<channel id="volumeDb" typeId="volumeChannelDb" />
+			<channel id="mute" typeId="muteChannel" />
+			<channel id="setInputSource" typeId="setInputSourceChannel" />
+			<channel id="displayInformation" typeId="displayInformationChannel" />
+		</channels>
+
+		<config-description>
+			<parameter name="address" type="text" required="true" >
+				<context>network-address</context>
+				<label>Address</label>
+				<description>The address of the Pioneer AVR to control.</description>
+			</parameter>
+			<parameter name="tcpPort" type="decimal" max="65535" min="1">
+				<default>23</default>
+				<label>TCP Port</label>
+				<description>The TCP port number used to connect to the AVR.</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+	
+	<!-- AVR control over Serial -->
+	<thing-type id="serialAvr">
+		<label>Pioneer AVR over Serial</label>
+		<description>Control a Pioneer AVR over through a Serial port (RS-232)</description>
+
+		<channels>
+			<channel id="power" typeId="powerChannel" />
+			<channel id="volumeDimmer" typeId="volumeChannelDimmer" />
+			<channel id="volumeDb" typeId="volumeChannelDb" />
+			<channel id="mute" typeId="muteChannel" />
+			<channel id="setInputSource" typeId="setInputSourceChannel" />
+			<channel id="displayInformation" typeId="displayInformationChannel" />
+		</channels>
+
+		<config-description>
+			<parameter name="serialPort" type="text" required="true" >
+				<label>Serial Port name</label>
+				<description>The Serial port name to use to connect to the AVR (Mandatory if Serial Protocol)</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="powerChannel">
+		<item-type>Switch</item-type>
+		<label>Power</label>
+		<description>Power ON/OFF the AVR</description>
+	</channel-type>
+
+	<channel-type id="volumeChannelDimmer">
+		<item-type>Dimmer</item-type>
+		<label>Volume</label>
+		<description>Increase/Decrease the volume (%) and mute/un-mute</description>
+		<category>Dimmer</category>
+		<state min="0" max="100" />
+	</channel-type>
+	
+	<channel-type id="volumeChannelDb">
+		<item-type>Number</item-type>
+		<label>Volume</label>
+		<description>Set the volume level (dB)</description>
+		<category>SoundVolume</category>
+		<state min="-80" max="12" step="0.5" />
+	</channel-type>
+
+	<channel-type id="muteChannel">
+		<item-type>Switch</item-type>
+		<label>Mute</label>
+		<description>Enable/Disable Mute on the AVR</description>
+	</channel-type>
+
+	<channel-type id="setInputSourceChannel">
+		<item-type>String</item-type>
+		<label>Input source</label>
+		<description>Select the input source of the AVR</description>
+		<state pattern="[0-9]{2}">

```

Comment:
The pattern is meant for UIs, which would display the option texts and not the keys, so probably better remove the pattern here.